src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Figure {
+    double getArea();
```

Comment:
```suggestion
    private String color;
```

create constructor and getColor() method also